src/util/resolve_test.ts

```diff
@@ -0,0 +1,31 @@
+import * as test from 'tape'
+import * as sinon from 'sinon'
+import { resolveSID } from './resolve'
+import Api from '../api/mock'
+
+const instances = [{ hash: 'instancehash', serviceHash: 'servicehash' }]
+const services = [{ hash: 'servicehash', sid: 'servicesid' }]
```

Comment:
This test have 2 services with same sid:
```suggestion
const services = [{ hash: 'servicehash', sid: 'servicesid' }, { hash: 'servicehash2', sid: 'servicesid' }]
```

Reply:
done